README.md

```diff
@@ -108,6 +111,30 @@ Report issues for this extension in the [ocds-extensions repository](https://git
 
 ## Changelog
 
+### 2023-11-13
+
+* Clarify semantics:
+  * Replace Finance.financeType (financeType.csv) with `Finance.financingPartyType` (`financingPartyType.csv`) and `Finance.type` (`financingArrangementType.csv`). Notably:
+    * "publicBondIssue" is replaced by 'debt' in `financingArrangementType.csv` and 'bond' `assetClass.csv`
+    * "supplierCredit" is replaced by 'vendor' in the `financingPartyType.csv` codelist
+  * Replace Finance.financeCategory (financeCategory.csv) with `Finance.assetClass` (`assetClass.csv`), `Finance.type` (`financingArrangementType.csv`) and `Finance.repaymentPriority` (`debtRepaymentPriority.csv`). Notably:
+    * "equity" is replaced by 'equity' in `assetClass`
+    * "seniorDebt" is replaced by 'debt' in `assetClass.csv` and 'senior' in `debtRepaymentPriority.csv`
+    * "mezzanineDebt" is replaced by 'debt' and 'equity' in `assetClass.csv` and 'subordinated' in `debtRepaymentPriority.csv`
+    * "grant" is replaced by 'grant' in `financingArrangementType.csv`
+    * "guarantee" is replaced by 'guarantee' in `financingArrangementType.csv`
+* Add fields:
+  * `Finance.resultsBased`
```

Comment:
Use alphabetical order

Reply:
Done in https://github.com/open-contracting-extensions/ocds_finance_extension/pull/26/commits/73c7856e8130bd1dcdb636823d5b55b2204b2e3c